ebean-core/src/main/java/io/ebeaninternal/server/type/ScalarTypeJsonObjectMapper.java

```diff
@@ -66,6 +122,19 @@ public boolean isJsonMapper() {
     public String jsonMapper(Object value) {
       return formatValue(value);
     }
+    
+
+
+
+    
+    @Override
+    public MutableHash createMutableHash(String json) {
+      if (false) { // TODO should we make that configurable?
```

Comment:
I think we could specify via the `@DbJson` annotation if the source json content should be kept or not.

`@DbJson(keepSource=true)`
MyBean myBean;

... then this method would move to the BeanProperty (well BeanPropertyJsonMapper)